config/prow/k8s-testgrid/k8s-testgrid.yaml

```diff
@@ -88,7 +87,6 @@ dashboards:
 - name: knative-extensions-release-1.0
 - name: knative-extensions-release-1.1
 - name: knative-extensions-release-1.10
-- name: knative-extensions-release-1.11
```

Comment:
is this line removal intentional?

Reply:
It's autogenerated. I believe it is - 1.11 was the last net-http01 release so dropping the jobs as well.

Reply:
oh, got it. Thanks